spec/vehicle_factory_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+RSpec.describe VehicleFactory do
+
+  it 'exists' do
+    factory = VehicleFactory.new
+
+    expect(VehicleFactory.new).to be_an_instance_of(VehicleFactory)
+  end
+
+  it 'also exists' do
+    factory = VehicleFactory.new
+    wa_ev_registrations = DmvDataService.new.wa_ev_registrations
+    
+    expect(wa_ev_registrations = DmvDataService.new.wa_ev_registrations).to be_an_instance_of(Array)
+  end
+
+  it 'takes the registered vehicles and arrays it into a Vehicle' do
+    factory = VehicleFactory.new
+    wa_ev_registrations = DmvDataService.new.wa_ev_registrations
+
+    expect(factory.create_vehicles(wa_ev_registrations)).to be_an_instance_of(Array)
+  end
```

Comment:
Apply the same feedback I gave on the FAcilityFactory spec to this spec too. It'd be best to go into those arrays and test what they're made up of.